perfkitbenchmarker/timing_util.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities for generating timing samples."""
+
+from contextlib import contextmanager
+import time
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import flags_validators
+from perfkitbenchmarker import sample
+
+
+# Name of the measurements configuration flag as a user would type it.
+MEASUREMENTS_FLAG_NAME = 'timing_measurements'
+# Valid options that can be included in the flag's list value.
+MEASUREMENTS_NONE = 'NONE'
```

Comment:
Please make the values lower case. We haven't been 100% consistent on this, but that's the preferred format.
Please document what each flag value means.